payjoin-cli/src/main.rs

```
@@ -60,12 +62,17 @@ async fn main() -> Result<()> {
         Commands::Send { bip21, fee_rate } => {
             app.send_payjoin(bip21, *fee_rate).await?;
         }
-        Commands::Receive { amount, .. } => {
-            app.receive_payjoin(*amount).await?;
+        Commands::Receive { amount, max_fee_rate, .. } => {
+            let fee_rate = max_fee_rate
+                .or(cli.max_fee_rate)
+                .or(config_max_fee_rate)
+                .unwrap_or(FeeRate::from_sat_per_vb(1).unwrap());
```

Should we error out if we recieve a invalid fee rate? Defaulting to 1 sat/vb sounds like it could result in unintended consequences esp. if it overwrites the existing session context max feerate

---

I get your point now, this was an overkill